superset/datasets/schemas.py

```diff
@@ -71,7 +71,7 @@ class DatasetMetricsPutSchema(Schema):
     metric_name = fields.String(required=True, validate=Length(1, 255))
     metric_type = fields.String(allow_none=True, validate=Length(1, 32))
     d3format = fields.String(allow_none=True, validate=Length(1, 128))
-    verbose_name = fields.String(allow_none=True, Length=(1, 1024))
+    verbose_name = fields.String(allow_none=True, metadata={Length: (1, 1024)})
```

Comment:
Same here, the original code is broken and you want:

```suggestion
    verbose_name = fields.String(allow_none=True, validate=Length(1, 1024))
```